examples/cra-ts-essentials/.storybook/main.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 
 module.exports = {
-  stories: ['../src/**/*.stories.@(tsx|mdx)'],
+  stories: ['../src/**/*.stories.tsx', '../src/**/Button.stories.mdx'],
```

Comment:
Isn't the pre-change code more comprehensive? I don't understand that you changed this part.